ifs.py

```diff
@@ -34,168 +34,199 @@
 import os
 
 # Constants
-Rd = 287. # Gas constant for dry air (J/K/kg)
-Rv = 461. # Gas constant for water vapour (J/K/kg)
-cp = 1006. # Heat capacity of air at constant pressure (J/K/kg)
-g = 9.81 # Accelleration due to gravity (m/s)
-a = 6371000. # Radius of earth (m)
+Rd = 287.  # Gas constant for dry air (J/K/kg)
+Rv = 461.  # Gas constant for water vapour (J/K/kg)
+cp = 1006.  # Heat capacity of air at constant pressure (J/K/kg)
+g = 9.81  # Accelleration due to gravity (m/s)
+a = 6371000.  # Radius of earth (m)
 Omega = 7.292e-5
 
+
 def get_reference_levels(levs):
     """ Return the reference pressure for level index coordinates."""
     thisdir = os.path.dirname(__file__)
-    l137 = pd.read_csv(thisdir+'/ecmwf_l137.txt', index_col=0)
+    l137 = pd.read_csv(thisdir + '/ecmwf_l137.txt', index_col=0)
     return l137.z[levs].values
 
+
 def pressure_funcs(ph, levs, z_axis=1):
     """ Variables related to the vertical discretization in IFS. """
     # Move z_axis to front
     _ph = np.moveaxis(ph, z_axis, 0)
 
-    if levs[0]<levs[1]: # IFS upside down format
+    if levs[0] < levs[1]:  # IFS upside down format

```

Comment:
```suggestion
    if levs[0] < levs[1]:

```
I don't think this comment is necessary